installbuilder/datafiles/Base_DSC.data

```diff
@@ -169,14 +169,16 @@ EOF
 
 chmod u+x $tmpfile
 $tmpfile 1> /dev/null 2> /dev/null
-if [ $? -ne 0 ]; then
+tmpfile_ctypes=$?
+rm $tmpfile
+
+python -c "import ctypes" 1> /dev/null 2> /dev/null
```

Comment:
So the script is tied specifically to `/usr/bin/python`, while line 175 is tied to anything in the current path.

I'm not sure if DSC has other dependencies on python being in `/usr/bin`. You'd need to peruse the source code, or ask the DSC team.

My question, though: What's the point of checking both the script form and the command line form? Is there a situation where the command line form (at line 175) might pass where the script form (at line 171) would not?

There situations where 175 would work where 171 would not (specifically python being around, with ctypes installed, but not in /usr/bin).

What's the purpose of two forms of checking? Is DSC dependent on a python in /usr/bin? If so, then what's the purpose of 175?